services/metadata/frontend/project-page/ProjectPage.svelte

```diff
@@ -68,101 +79,102 @@
   </div>
 {/if}
 
-<div class="container" in:fade={{duration: 200}}>
-  {#if mobileResolution}
-    <button on:click={() => history.back()} class=goback-button title="go back to the projects list" disabled={!$previousRoute && window.history.length <= 2}>
-      <svg class=icon fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10 19l-7-7m0 0l7-7m-7 7h18" />
-      </svg>
-      <span class=button-label>Go Back</span>
-    </button>
-  {/if}
-  <div class="row" style="flex-wrap: wrap;">
-    <h1 class="title top-heading">
-      {$currentProject?.name}
-    </h1>
-    {#if $currentProject?.alternateName}
-    <div class="row">
-      <h4 class="title new-title">
-        Also known as:&nbsp;
-        <span style="color:var(--secondary-colour)">{$currentProject?.alternateName.join(", ")}</span>
-      </h4>
-    </div>
+{#if $currentProject}
```

Comment:
I think instead of using `onMount`, you could do this here:

```svelte
{#await getProjectMetadata()}
  <div> ...
{:else}
  <Loading />
{/await}
```

I'm not sure it's actually better... but it might be cleaner than blocking `onMount` with an `await`?

Reply:
It isn't valid block of code. `Await` has no 'else` so `if` would need to stay anyway, apart of it won't work anyway ;)